framework/src/base/MooseApp.C

```diff
@@ -170,6 +170,7 @@ MooseApp::MooseApp(const std::string & /*deprecated_name*/, InputParameters para
     _check_input(getParam<bool>("check_input")),
     _restartable_data(libMesh::n_threads())
 {
+  srand(processor_id());
```

Comment:
No   ?

@aeslaughter 's commit message says "Remove srand"... but then it's still in his commit.